src/models/BrowserStyleSheet.js

```diff
@@ -17,88 +17,77 @@
  * Note: replace · with * in the above snippet.
  * */
 import extractCompsFromCSS from '../utils/extractCompsFromCSS'
+import stringifyRules from '../utils/stringifyRules'
 import getNonce from '../utils/nonce'
 import type { Tag } from './StyleSheet'
 import StyleSheet, { SC_ATTR, LOCAL_ATTR } from './StyleSheet'
 
-export const COMPONENTS_PER_TAG = 40
+declare var __DEV__: ?string
 
-class BrowserTag implements Tag {
-  isLocal: boolean
-  components: { [string]: Object }
-  size: number
-  el: HTMLElement
-  ready: boolean
-
-  constructor(el: HTMLElement, isLocal: boolean, existingSource: string = '') {
-    this.el = el
-    this.isLocal = isLocal
-    this.ready = false
-
-    const extractedComps = extractCompsFromCSS(existingSource)
-
-    this.size = extractedComps.length
-    this.components = extractedComps.reduce((acc, obj) => {
-      acc[obj.componentId] = obj // eslint-disable-line no-param-reassign
-      return acc
-    }, {})
-  }
+const DISABLE_SPEEDY =
+  (typeof __DEV__ === 'boolean' && __DEV__) ||
```

Comment:
it'd be nice if there was a way to turn SPEEDY on without `NODE_ENV=production` for easy testing

Reply:
Well, I don’t think that should be necessary as it should stay an implementation detail at all costs. Also, if this’d be dynamic both BrowserTags end up in the user’s final bundle